backend/game/client_handler.py

```diff
@@ -0,0 +1,48 @@
+import json
+import logbook
+from tornado.websocket import WebSocketHandler
+
+from .core import Game
+from .. import config
+
+class ClientWebSocketHandler(WebSocketHandler):
+
+    game = Game()
+    message_handlers = dict()
```

Comment:
This should have a more indicative name in order to understand the meaning of this variable